templates/base.html

```diff
@@ -0,0 +1,87 @@
+{% load static %}
+<!DOCTYPE html>
+<html>
+<head>
+	 <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <!-- Bootstrap CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+  <link rel="stylesheet" type="text/css" href="{% static 'css/main.css' %}">
+  {% if title %}
+  	<title> Falcon - {{title}}</title>
+  {% else %}
+  	<title> Falcon </title>
+  {% endif %}
+</head>
+<body>
+	<header class="site-header">
+	  <nav class="navbar navbar-expand-md navbar-dark bg-steel fixed-top">
+	    <div class="container">
+	      <a class="navbar-brand mr-4" href="/"> Falcon </a>
+	      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarToggle" aria-controls="navbarToggle" aria-expanded="false" aria-label="Toggle navigation">
+	      <span class="navbar-toggler-icon"></span>
+	      </button>
+	      <div class="collapse navbar-collapse" id="navbarToggle">
+	        <div class="navbar-nav mr-auto">
+	          <a class="nav-item nav-link" href="/">Home</a>
+	          <a class="nav-item nav-link" href="#">About</a>
+	        </div>
+	        <!-- Navbar Right Side -->
+	        <div class="navbar-nav">
+            {% if user.is_authenticated %}
+              {% if user.Role == Manager %}
+                <a class="nav-item nav-link" href="#">New Task</a>
+                <a class="nav-item nav-link" href="{% url 'logout' %}">Logout</a>
+              {% else %}
+                <a class="nav-item nav-link" href="{% url 'logout' %}">Logout</a>
+              {% endif %}
+            {% else %}
+	          <a class="nav-item nav-link" href="{% url 'login' %}">Login</a>
+            {% endif %}
+	        </div>
+	      </div>
+	    </div>
+	  </nav>
+	</header>
+
+<main role="main" class="container">
+  <div class="row">
+    <div class="col-md-8">
+      {% if messages %}
+         {% for message in messages %}
+           <div class="alert alert-{{ message.tags }}">
+            {{ message }}
+           </div>
+         {% endfor %}
+      {% endif %}
+      {% block content %}{% endblock %}
```

Comment:
I would be glad for further explanation here.

Reply:
It gives you such an error message if you enter a password or a non-existent user..do Google

Reply:
ok thanks